.circleci/config.yml

```diff
@@ -89,26 +97,22 @@ jobs:
 
   frontend-tests:
     docker:
-      - image: greenpeaceinternational/p4-unit-tests:node20.18.0
+      - image: cimg/node:20.18.3
```

Comment:
Do you think if there isn't a problem if we're currently using `20.18.0`?

Reply:
I just picked the latest `20.18`. Where do you mean we use `20.18.0`? On the develop environment we just have `20` for nvm, so it should pick the latest minor release available

Reply:
For example [here](https://github.com/greenpeace/planet4-circleci/blob/main/config.default#L18) but maybe I'm wrong and that is not a problem!

Reply:
Ah good catch. Minor releases shouldn't contain breaking changes, but just to be safe I switched all references to `20.18.0`